app/waf/src/sequence.ts

```diff
@@ -32,6 +37,9 @@ export class MySequence implements SequenceHandler {
     let result: object = {};
     try {
       const {request, response} = context;
+
+      //await this.authRequest(request);
```

Comment:
Still need this?

Reply:
Yes. If we decide to integrate with OpenStack, we have to auth every user's request before handling it.

Reply:
It should also be wrapped with `if (PRODUCT_RELEASE) ...`, I think.